examples/annotation_import/video.ipynb

```diff
@@ -69,6 +69,7 @@
       "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.      global_keys=global_key, # A paginated collection of data row objects
</pre><p><code>global_keys=[global_key]</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1006/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>